common/views/components/WorkTitle/WorkTitle.tsx

```diff
@@ -0,0 +1,5 @@
+const WorkTitle = ({ title }: { title: string }) => (
```

Comment:
Could this just be HTMLBlock or something more generic>?

Reply:
It could, I just wanted to make it clear we should be using this when rendering the work titles

Reply:
👍 I suppose it's also something we don't really want to be permitting.